packages/combobox/primitives.tsx

```diff
@@ -49,7 +49,9 @@ export function ComboboxPopover({
   )
 }
 
-export interface ComboboxInputProps extends ReachComboboxInputProps {
+type InputProps = InputHTMLAttributes<HTMLInputElement> &
+  ReachComboboxInputProps
+export interface ComboboxInputProps extends InputProps {
```

Comment:
@BRKalow  - Wanted to get your take on the types here. 

I think I need to potentially split this once more (maybe with `Omit`?) so that I can make [`onChange` optional here at the top level (the default export, `<Combobox />`)](https://github.com/hashicorp/react-components/pull/289/files#diff-2d04980ac40707a6752abd67b8b92f569f0582630476c1c3d89881286419a52fR22) since there's a chance it's not necessary if folks only require `onSelect` for their use case

Reply:
From: https://stackoverflow.com/questions/43159887/make-a-single-property-optional-in-typescript

```ts
type Optional<T, K extends keyof T> = Pick<Partial<T>, K> & Omit<T, K>;

// ...

Optional<ComboboxInputProps, 'onChange'>
```

Or just make it always optional and provide an empty fn as a default:
```
onChange: () => {}
```